Code_Review_Assignment_JrSDET.java

```diff
@@ -1 +1,172 @@
+package com.org.happyfox;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.util.concurrent.TimeUnit;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.interactions.Actions;
+
+public class Testcase101 {
+
+    public static void main(String[] args) throws InterruptedException, AWTException {
+        System.setProperty("webdriver.gecko.driver", "C:\\Users\\Johny\\Downloads\\geckodriver-v0.33.0-win64\\geckodriver.exe");
+        WebDriver driver = new FirefoxDriver();
+        driver.get("https://interview.supporthive.com/staff/");
+        driver.manage().timeouts().implicitlyWait(30, TimeUnit.SECONDS);
+        driver.manage().window().maximize();
+        driver.findElement(By.id("id_username")).sendKeys("Agent");
+        driver.findElement(By.id("id_password")).sendKeys("Agent@123");

```

Comment:
Avoid using hardcoded values like  "Agent", "Agent@123". These should be extracted into constants or configuration files.